docs/mllib-feature-extraction.md

```diff
@@ -506,7 +523,7 @@ v_N
 
 [`ElementwiseProduct`](api/scala/index.html#org.apache.spark.mllib.feature.ElementwiseProduct) has the following parameter in the constructor:
```

Comment:
Not part of your PR, but this API doc link would be better if it were inside the codetabs